app/src/lib/editors/win32.ts

```diff
@@ -12,7 +12,7 @@ import { IFoundEditor } from './found-editor'
 
 interface IWindowsAppInformation {
   displayName: string
-  publisher: string
+  publisher: string | string[]
```

Comment:
This is what Desktop retrieves from the registry, so it should _always_ be just one publisher.
```suggestion
  publisher: string
```